src/adservice/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
```

Comment:
I'm not able to understand what the 2 adservice/ changes achieve.
Are they changing anything? (I'm illiterate in Java.)

Reply:
removed, my vscode populated those